src/pymatgen/io/cif.py

```diff
@@ -969,8 +969,22 @@ def _get_structure(
         primitive: bool,
         symmetrized: bool,
         check_occu: bool = False,
+        min_vol: float = 0.5,
```

Comment:
I hope 0.5 Å³ would be a good default threshold volume to consider a structure unphysical, comment/input welcome